browsers/chrome/manifest.json

```diff
@@ -42,6 +42,13 @@
         "css": ["public/css/noatb.css"],
         "run_at": "document_start"
       },
+      {
+        "js": [
+            "public/js/content-scripts/settings-toggle.js"
+        ],
+        "matches": ["https://duckduckgo.com/app?post=1*"],
```

Comment:
Inject content script into post install page. When testing, make sure to point this at your box.